trio/_socket.py

```diff
@@ -310,13 +293,12 @@ def fromfd(*args, **kwargs):
 if hasattr(_stdlib_socket, "fromshare"):
 
     @_wraps(_stdlib_socket.fromshare, assigned=(), updated=())
-    @_add_to_all
+    # @_add_to_all
```

Comment:
@jmfrank63  in general, please be sure that new pushes are addressing outstanding comments.  Otherwise it seems that github automatically marks the comments as resolved, and they git lost.  I think this was a pervasive communication problem on your last PR.

Reply:
@belm0 Thanks for letting me know, I will go through all of them and address them in the next push. There is so much to learn, thanks for the patience. I will also try to attempt squashing this time, as there is so much back and forth.

Reply:
Thank you for considering.  (And again it's dubious that GitHub behaves that way.)

About squashing, that can be done trivially by the person doing the final merge into master.